### DISCUSSION TITLE
Mobi Display

### DISCUSSION THREAD

**Discussion by Salih-Benamara** [Open-Systems-Pharmacology/Forum#1150] [Questions & Problems] [answered]
Dear community,

Hope you are doing well :D I would like to ask for your help regarding the Mobi interface, I can't get the complete interface for Building Blocks or Simulations. 
I think that the  problem is related to the display because the files are loaded on Mobi but there is nothing that is displayed  
what I’ve done so far to try to solve the problem : 

1/  Set the display parameters on the '' Views'' tab 
2/  Uninstall and reinstall the program 

Thanks ^^
Salih

![Probleme 2  MOBI](https://user-images.githubusercontent.com/100346781/155502542-d5706565-0511-4129-9a52-85b8c4d592f2.JPG)
![Probleme MOBI](https://user-images.githubusercontent.com/100346781/155502545-adec1cc2-8242-436f-9890-a7c3c68bf9ff.JPG)

**Comment by Yuri05** [comment, marked as answer]
- Close MoBi.
- In Windows Explorer: go to the folder **C:\Users\\_USERID_\\AppData\Roaming\Open Systems Pharmacology\MoBi\10.0** 
    (replace **_USERID_** with your windows user name)
- Delete the file **Settings.xml**
- Start MoBi

**Reply by Salih-Benamara** [reply]
Hi, thanks !

I did it and it Works Now 

Thanks a lot !

**Reply by Mosaic815** [reply]
I am using the MoBi v12 and the whole OSP-Suite package was installed and saved in the Program files path. How then do I follow your instruction? Nevertheless, I searched this " C:\Users\USERID\AppData\Roaming\Open Systems Pharmacology\MoBi\12.0" with the USERID replaced with my windows user name but had no results. I went head regardless to check this path: "C:\Program Files\Open Systems Pharmacology\MoBi 12.0" and I did not see "Settings.xml" to delete. 

Please consider that I am a beginner to all these whiles providing any useful instructions to my case. Thank you in advance.

**Reply by Mosaic815** [reply]
It worked. Thank you.

**Comment by StephanSchaller** [comment]
@Yuri05 , this still happens in V11, is this sth that needs fixing or what can be done here?

**Reply by msevestre** [reply]
The reason is unclear (looks like it has something to do with docking/undocking). The fix is simple. Adding an option to reset the layout. 
Or do not save the layout and always start as intended

**Reply by CECI-LALA** [reply]
Need more details about "adding an option", thank you. 
I can't find the "reset the layout" button in UTILITIES OR VIEWS in MoBi.
 "reset the layout" button just can be find in PK-Sim.

**Reply by StephanSchaller** [reply]
@CECI-LALA , the comment from Michael was just an "internal" comment for the software developers. For a solution, pleaser refer to the answers above
